packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+%exclude %{_cross_unitdir}/system-update-cleanup.service
+%exclude %{_cross_unitdir}/system-update-pre.target
+%exclude %{_cross_unitdir}/system-update.target
+%exclude %{_cross_unitdir}/systemd-update-done.service
+
+%dir %{_cross_libdir}/udev
+%{_cross_libdir}/udev/ata_id
+%{_cross_libdir}/udev/cdrom_id
+%{_cross_libdir}/udev/dmi_memory_id
+%{_cross_libdir}/udev/fido_id
+%{_cross_libdir}/udev/mtd_probe
+%{_cross_libdir}/udev/scsi_id
+%exclude %{_cross_libdir}/udev/v4l_id
+
+%dir %{_cross_udevrulesdir}
+%{_cross_udevrulesdir}/50-udev-default.rules
+%{_cross_udevrulesdir}/60-autosuspend.rules
+%{_cross_udevrulesdir}/60-block.rules
+%{_cross_udevrulesdir}/60-cdrom_id.rules
+%{_cross_udevrulesdir}/60-drm.rules
+%{_cross_udevrulesdir}/60-evdev.rules
+%{_cross_udevrulesdir}/60-fido-id.rules
+%{_cross_udevrulesdir}/60-infiniband.rules
+%{_cross_udevrulesdir}/60-input-id.rules
+%{_cross_udevrulesdir}/60-persistent-input.rules
+%{_cross_udevrulesdir}/60-persistent-storage-tape.rules
+%{_cross_udevrulesdir}/60-persistent-storage.rules
+%{_cross_udevrulesdir}/60-sensor.rules
+%{_cross_udevrulesdir}/60-serial.rules
+%{_cross_udevrulesdir}/64-btrfs.rules
+%{_cross_udevrulesdir}/70-memory.rules
+%{_cross_udevrulesdir}/70-power-switch.rules
+%{_cross_udevrulesdir}/75-net-description.rules
+%{_cross_udevrulesdir}/75-probe_mtd.rules
+%{_cross_udevrulesdir}/80-drivers.rules
+%{_cross_udevrulesdir}/80-net-setup-link.rules
+%{_cross_udevrulesdir}/81-net-dhcp.rules
+%{_cross_udevrulesdir}/99-systemd.rules
+
+
+# Exclude desktop-related device rules.
+%exclude %{_cross_udevrulesdir}/60-persistent-alsa.rules
+%exclude %{_cross_udevrulesdir}/60-persistent-v4l.rules
+%exclude %{_cross_udevrulesdir}/70-camera.rules
+%exclude %{_cross_udevrulesdir}/70-joystick.rules
+%exclude %{_cross_udevrulesdir}/70-mouse.rules
+%exclude %{_cross_udevrulesdir}/70-touchpad.rules
+%exclude %{_cross_udevrulesdir}/70-uaccess.rules
+%exclude %{_cross_udevrulesdir}/71-seat.rules
+%exclude %{_cross_udevrulesdir}/73-seat-late.rules
+%exclude %{_cross_udevrulesdir}/78-sound-card.rules
+
+%dir %{_cross_sysusersdir}
+%{_cross_sysusersdir}/basic.conf
+%{_cross_sysusersdir}/systemd-journal.conf
+
+%dir %{_cross_tmpfilesdir}
+%{_cross_tmpfilesdir}/etc.conf
+%{_cross_tmpfilesdir}/home.conf
+%{_cross_tmpfilesdir}/journal-nocow.conf
+%{_cross_tmpfilesdir}/provision.conf
+%{_cross_tmpfilesdir}/static-nodes-permissions.conf
+%{_cross_tmpfilesdir}/systemd-pstore.conf
+%{_cross_tmpfilesdir}/systemd-tmp.conf
+%{_cross_tmpfilesdir}/systemd.conf
+%{_cross_tmpfilesdir}/tmp.conf
+%{_cross_tmpfilesdir}/var.conf
+%exclude %{_cross_tmpfilesdir}/x11.conf
+
+%{_cross_datadir}/dbus-1/services/org.freedesktop.systemd1.service
+%{_cross_datadir}/dbus-1/system.d/org.freedesktop.login1.conf
+%{_cross_datadir}/dbus-1/system.d/org.freedesktop.systemd1.conf
+%exclude %{_cross_datadir}/dbus-1/system-services
+%exclude %{_cross_datadir}/factory/etc/vconsole.conf
```

Comment:
`vconsole.conf` should go with the other `%{_cross_factorydir}%{_cross_sysconfdir}` excludes.